assets/company-reports/components/ContentActivity.tsx

```diff
@@ -77,12 +121,12 @@ class ContentActivity extends React.Component<any, any> {
         };
     }
 
-    componentWillMount() {
+    UNSAFE_componentWillMount() {
         // Fetch the genre & company aggregations to populate those dropdowns
         this.props.fetchAggregations(REPORTS['content-activity']);
     }
 
-    componentWillReceiveProps(nextProps: any) {
+    UNSAFE_componentWillReceiveProps(nextProps: any) {
```

Comment:
here it seems like it's doing some derived stated, would be good to [avoid it](https://legacy.reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#what-about-memoization)

Reply:
yeah, make sense. I'll do the adjustments.

Reply:
done